pkg/core/state/notification_event.go

```diff
@@ -95,6 +168,10 @@ func (aer *AppExecResult) EncodeBinaryWithContext(w *io.BinWriter, sc *stackitem
 		aer.Events[i].EncodeBinaryWithContext(w, sc)
 	}
 	w.WriteVarBytes([]byte(aer.FaultException))
+	w.WriteVarUint(uint64(len(aer.Invocations)))
```

Comment:
Regarding DB compatibility: I'm thinking about adding `SaveInvocations` to `dao.Version` because: if the node was running with `SaveInvocatoins: true` and then it was switched to `SaveInvocations: false`, then currently there's no way from the user side to distinguish missing invocatoins from empty invocations, and it seems like a broken DB for me. I propose to explicitly return an error from the `(bc *Blockchain) init()` if `SaveInvocations` was changed without DB resync.

@roman-khimov, ACK?

Reply:
Makes sense, follows https://github.com/nspcc-dev/neo-go/blob/master/docs/node-configuration.md#db-compatibility (don't forget to update it).